src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -174,6 +174,6 @@ def __init__(self):
 
 DEFAULT_MAXIMUM_INSTANCE_COUNT = 100
 
-DEPLOYMENT_STORAGE_AUTH_TYPES = ['SystemAssignedIdentity', 'UserAssignedIdentity', 'StorageAccountConnectionString']
+DEPLOYMENT_STORAGE_AUTH_TYPES = ['StorageAccountConnectionString']
```

Comment:
Are we not supporting 'SystemAssignedIdentity', 'UserAssignedIdentity' anymore?

Reply:
Not for this release, we will support it in May one.